test/notifications/notification_test.exs

```diff
@@ -194,30 +195,72 @@ defmodule Notifications.NotificationTest do
           end_time: 19
         })
 
-      bridge_unexpired =
+      # Due to the blackout logic in get_or_create_from_bridge_movement, we
+      # have to rig the inserted_at timestamps of these bridge movements for
+      # the test to work the way we would expect.
+
+      bridge_lowered_unexpired =
         Notification.get_or_create_from_bridge_movement(%{
           status: :lowered,
           lowering_time: nil,
-          created_at: baseline_time - eight_hours + 1
+          created_at: baseline_time - eight_hours + 1234
         })
 
-      filled_in_bridge_unexpired = %Notification{
-        bridge_unexpired
-        | start_time: baseline_time - eight_hours + 1,
-          end_time: baseline_time + 1,
+      Skate.Repo.query!(
+        "UPDATE bridge_movements SET inserted_at = inserted_at - interval '1 hour'"
```

Comment:
Oh huh I didn't realize Postgres let you do stuff like `interval '1 hour'`, that looks handy!

Reply:
Postgres can do _tons_ of stuff, it's always worth a look at the function reference if you want to do something fancy. The odds of finding something useful are good.